Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -131,7 +131,7 @@ extension File {
         let fileRegions = regions()
         let violatingRanges = violatingRanges.filter { range in
             let region = fileRegions.filter {
-                $0.contains(Location(file: self, offset: range.location))
+                $0.contains(Location(file: self, characterOffset: range.location))
```

Comment:
That is not *character* more accurately.
But that is not only here. :wink:
It may hard to keep consistency without using correct parameter types such as `String.UTF16Index`